### PR TITLE
Fix bonniemat 1106

### PR DESCRIPTION
**Pull request by dczulada**
A patient that meets the DENEX should not have observation_values
https://oncprojectracking.healthit.gov/support/browse/BONNIEMAT-1106

**Submitter:**
- [x] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/helpers/calculator_helpers.js [hunk 1]
```diff
@@ -132,6 +132,9 @@ module.exports = class CalculatorHelpers {
       if ('DENEXCEP' in populationResults) {
         populationResultsHandled.DENEXCEP = 0;
       }
+      if ('observation_values' in populationResults) {
+        populationResultsHandled.observation_values = [];
+      }
```
Comment (adongare) [inline lib/helpers/calculator_helpers.js:137]: do we need to conder multiple IPP scenario? I believe the ratio measures can have multiple IPPs.

Comment (dczulada) [reply lib/helpers/calculator_helpers.js:137]: Yes, multiple IPP scenarios do need to be considered.

The "createEpisodePopulationValues" method calls back to the "handlePopulationValues" method for each Episode of Care.  The fix above should address the results for each episode as well as the patient totals.

https://github.com/projecttacoma/cqm-execution/blob/9ce4a73874194157990ca088209906d19d7889c8/lib/helpers/calculator_helpers.js#L305

Comment (adongare) [reply lib/helpers/calculator_helpers.js:137]: thanks for the clarification and fix!

**Review by adongare — APPROVED** [review]

**Review by luis1van — APPROVED** [review]
